src/zope/app/apidoc/zcmlmodule/browser.py

```diff
@@ -32,6 +31,8 @@
 
 class Menu(object):
     """Menu View Helper Class"""
+    context = None
+    request = None
```

Comment:
They are. Plus I think it's just weird to see an object accessing attributes that it never sets. I know there's some magic going on with the publisher that sets them. I'd prefer that they have explicit constructors, which I think the publisher *also* supports, but I've been making the minimal change to shut the warnings up.